src/cli/command.js

```diff
@@ -53,30 +53,44 @@ Command.prototype.collectUnknownOptions = function () {
   this.allowUnknownOption();
   this.getUnknownOptions = function () {
     const opts = {};
+
+    /**
+     * Commander.js already presents the unknown args split by "=",
+     * and shorthand switches already split, like -asd => -a, -s, -d
+     */
     const unknowns = this.unknownArgv();
 
+    const singleFlagArgs = unknowns.filter((flag) => {
+      return flag.match(/^-[a-zA-Z0-9]$/);
```

Comment:
Possible edge case: `--config=-v`.  In practice..can't think of any scenario that run into this, but thought I'd make a note

Reply:
You're right, this will be filtered, caught and stopped. 

I'm trying to think of a nicer way to interpret ambiguous arguments :/